taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.validators import RegexValidator
+
+from taxi.models import Driver, Car
+
+
+DRIVER_LICENSE_VALIDATOR = RegexValidator(
+    regex=r"^[A-Z]{3}[\d]{5}$",
+    message="Enter a valid value license number:"
```

Comment:
Typographical Error: The message should use 'license' instead of 'value license' to maintain consistency and clarity.